devicon.json

```diff
@@ -355,7 +355,7 @@
                 "line-wordmark"
             ]
         },
-        "color": "#303284",
+        "color": "#ab0001",
```

Comment:
Same color as the main fill in the `original-wordmark` SVG
```suggestion
        "color": "#aa0001",
```

Reply:
Sure. Done